app/_hub/kong-inc/request-transformer/index.md

```diff
@@ -118,6 +118,8 @@ params:
   extra: |
     Note: if the value contains a `,` then the comma-separated format for lists cannot be used. The array notation must be used instead.
 
+    Note: the `X-Forwarded-*` are non-standard header fields written by Nginx to inform the upstream about client details and can't be overwritten by this plugin. If overwriting these header fields is needed, please see [post-function plugin in Serverless Functions](https://docs.konghq.com/hub/kong-inc/serverless-functions/).
```

Comment:
For readability, I think it would be better to combine the notes into one section, eg:

```suggestion
**Notes**: 
* If the value contains a `,` then the comma-separated format for lists cannot be used. The array notation must be used instead.
* The `X-Forwarded-*` fields are non-standard header fields written by Nginx to inform the upstream about client details and can't be overwritten by this plugin. If you need to overwrite these header fields, see the [post-function plugin in Serverless Functions](https://docs.konghq.com/hub/kong-inc/serverless-functions/).
```

Reply:
Thank you!